build.sh

```diff
@@ -235,37 +247,52 @@ crontab -l
 EOF
 
 
-ssh $osname  "$VM_SHUTDOWN_CMD"
+if [ -e "hooks/sysUpdate.sh" ]; then
+  echo "hooks/sysUpdate.sh"
+  cat "hooks/sysUpdate.sh"
+  ssh $osname sh<"hooks/sysUpdate.sh"
 
-sleep 30
+  # We need to reboot after system updates
+  restart_and_wait
+fi
 
-###############################################################
 
-if $vmsh isRunning $osname; then
-  if ! $vmsh shutdownVM $osname; then
-    echo "shutdown error"
-  fi
+if [ -e "hooks/postBuild.sh" ]; then
```

Comment:
please keep the "postBuild.sh" at the original position. It's expected to be executed at the first moment when vm is up.

Reply:
done